src/popover/popover.spec.ts

```diff
@@ -76,7 +76,7 @@ describe('ngb-popover', () => {
       directive.triggerEventHandler('click', {});
       fixture.detectChanges();
       expect(getWindow(fixture.nativeElement)).toBeNull();
-      expect(directive.nativeElement.getAttribute('aria-describedby')).toBeNull();
+      expect(directive.nativeElement.getAttribute('aria-describedby')).toBe('null');
```

Comment:
Hmm, this doesn't sound right. It seems like here we would have `aria-describedby="null"` while what we want is no attribute at all. Was this change necessary after migration to the new renderer?